### DISCUSSION TITLE
Turning threaded optimization on.

### DISCUSSION THREAD

**Discussion by xolostyak1337** [ppy/osu#27666] [Q&A] [answered]
osu std.
Recently I have encountered an issue with random lag spikes or frame drops. I have tried every singe method that could potentially solve the problem but nothing helped. Eventually I encountered the threaded optimization option in NVIDIA control panel and it greatly increased the game perfomance. The lag spikes are gone and the frames are higher than ever. But this option is forced off after restarting osu! Is there a way I can have it turned on by the default?

**Comment by peppy** [comment, marked as answer]
The closest you can get is turning on compatibility mode, which will bypass this setting.

We have it turned off because the game has major issues when it is turned on for a large portion of users.
